app/serializers/limited_notice_serializer.rb

```diff
@@ -0,0 +1,47 @@
+class NoticeSerializer < ActiveModel::Serializer
```

Comment:
Based on the filename I think you might want:
```ruby
 class LimitedNoticeSerializer < ActiveModel::Serializer
```
Seems like you might be redefining an existing class accidentally.